### ISSUE TITLE
[Bug]: SB8 UI, Search results on dark mode are tough to read

### ISSUE THREAD

**Issue by domyen** [storybookjs/storybook#26018]
### Describe the bug

The search results for SB8 UI dark mode are currently a shade of gray that's tough to read. I'd expect the title to be closer to to white.
<img width="357" alt="image" src="https://github.com/storybookjs/storybook/assets/263385/a6efcd8d-de64-4f82-a0b6-a7206d7ceee2">

### To Reproduce

1. Spin up an SB8 project (like [this one](https://6480e1b0042842f149cfd74c-lsmavabuym.chromatic.com/?path=/story/components-snapshotimage--taller-constrained))
2. Use the dark theme
3. Search for something
4. See the search result color

**Comment by sakit0** [comment]
Can I Have this issue?

**Comment by sash9696** [comment]
Hi  @domyen @vanessayuenn . Could u please assign this to me.

**Comment by valentinpalkovic** [comment]
Before someone starts on this, please let's wait on the feedback of some of our designers. 

cc @cdedreuille, @MichaelArestad

**Comment by cdedreuille** [comment]
Yes I believe the title should be white. Not sure what went wrong here when we recoded it.

**Comment by vanessayuenn** [comment]
@sakit0 @sash9696 we'd welcome a PR fixing this (changing the text to white as per @cdedreuille). 

Be sure to check the [contribution guide](https://storybook.js.org/docs/react/contribute/how-to-contribute) and hop onto the [Storybook Discord](https://discord.gg/storybook) if you need any help getting started. Looking forward to your contribution! ✨
